runtime/plugin/matchit.vim

```diff
@@ -1,8 +1,10 @@
 "  matchit.vim: (global plugin) Extended "%" matching
-"  Last Change: Fri Jul 29 01:20 AM 2016 EST
+"  Last Change: 2016 Aug 21
 "  Maintainer:  Benji Fisher PhD   <benji@member.AMS.org>
 "  Version:     1.13.2, for Vim 6.3+
 "		Fix from Tommy Allen included.
```

Comment:
That's because I hoped, that Benji would see that or react to it somehow. I even noticed him [directly](https://groups.google.com/d/msg/vim_use/iJV8-JWsQWE/RaHHIar-FAAJ) about your patch, but did not receive any feedback yet.